extern/libjpeg/jconfig.h

```diff
@@ -29,7 +29,7 @@ typedef unsigned char boolean;
 #ifdef JPEG_INTERNALS
 
 /* #undef RIGHT_SHIFT_IS_UNSIGNED */
-#define INLINE __inline__
+#define INLINE __inline
```

Comment:
This is kind of a dangerous change here. The idea is that the jpeg library is external to us: thus, we should not modify it. Why are you making this change?